backend/app/techniques/router.py

```
@@ -0,0 +1,201 @@
+
+            try:
+                technique = self._create_technique(definition)
+                items = TECHNIQUE_TO_ITEMS.get(tech_id, [])
+                primary_item = items[0] if items else "A1"
+                result = await technique.evaluate(state, primary_item)
+                results.append(result)
+
+                if on_progress:
+                    on_progress(i + 1, len(technique_ids), tech_id, result.success)
+
+            except Exception as e:
```

Ruff (0.14.14)

[warning] Do not catch blind exception: `Exception`

(BLE001)